prototypes/styletransfer/videos.py

```diff
@@ -30,6 +30,20 @@ def extract_frames_from_gif(gif_path: str) -> List[np.array]:
 
     return frames
 
+def save_frames_to_disk(images: List[np.array],
+                        directory: str):
+    if os.path.exists(directory): shutil.rmtree(directory)
+    os.makedirs(directory)
+
+    for i, image in enumerate(images):
+        Image.fromarray(image).save(os.path.join(directory, f"frame{i}.jpeg"))
+
+def make_gif(images: List[np.array], 
+             directory: str = "./transferred", 
+             gif_name: str = "gif.gif"):
+    save_frames_to_disk(images, directory)
+    subprocess.run(["ffmpeg", "-y", "-f", "image2", "-framerate", "15", "-i", f"{directory}/frame%d.jpeg", f"{gif_name}"])
```

Comment:
15 fps is too much. Not sure which number we should use. GIFs don't have the fps concept.